src/test/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSourceDeserializationTest.java

```diff
@@ -0,0 +1,56 @@
+package io.jenkins.plugins.gitlabbranchsource;
+
+import java.lang.reflect.Field;
+import jenkins.branch.BranchSource;
+import jenkins.scm.api.SCMSource;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.junit.Assert.assertNotNull;
+
+public class GitLabSCMSourceDeserializationTest {
+
+    private static final String PROJECT_NAME = "project";
+    private static final String SOURCE_ID = "id";
+
+    @Rule
+    public final RestartableJenkinsRule plan = new RestartableJenkinsRule();
+
+    @Test
+    public void afterRestartingJenkinsTransientFieldsAreNotNull() throws Exception {
+        plan.addStep(new Statement() {
```

Comment:
You can use `then` which accepts lambda :)

```suggestion
        plan.then(j -> {
```

Avoids the whole new Statement and overriding evaluate